Added forced stop

Avatat opened the pull request:
Related to: https://github.com/iocage/iocage/issues/532
I added force option to `iocage stop` command. It skip all pre-stop actions, kill jail process and do post-stop actions.

I don't know why git recognize so many lines as changed in `iocage/lib/ioc_stop.py`, because I added only few lines and indented many other lines.. I will be glad if someone explain me this - I'm new in git world :)
EDIT: Probably it is related to tab and space indentions.. How can I fix it? @skarekrow could you help me?

Avatat committed:
Added forced stop

Avatat committed:
Fixed Travis error

Fixed Travis error, related to previous commits.

Avatat committed:
Fixed Travis error

Avatat committed:
Revert "Added forced stop"

This reverts commit c74694a5e0c2e21c8a45ef0b329d96f4c9ae8a38.

Avatat committed:
Fixed indents

Avatat committed:
Fixed Travis errors

skarekrow reviewed (changes requested):
Looks good but a couple changes:
1 - force should be a kwarg and placed after the rest to minimize disturbance with existing API usage
2 - We should elaborate on the force help text describing what it's doing.

Avatat commented:
@skarekrow, I moved my changes to new branch (from my master branch) and created new PR: https://github.com/iocage/iocage/pull/553

I changed type of `force` arguments and added better help text.
